### DISCUSSION TITLE
关于Docker-compose部署的空间占用问题

### DISCUSSION THREAD

**Discussion by toyo2333** [DIYgod/RSSHub#13093] [Q&A]
### 这是一个什么样的功能？

我使用docker-compose部署的时候官方示例里搭配了browserless/chrome这个容器。**而docker run版本则没有**

问这个问题提是因为，browserless/chrome的容器占用非常大的空间，光镜像就有2.9G了（我VPS一共10G硬盘有点吃紧）。

我想知道，占用这么大正常么？是否包含browserless/chrome在功能上的区别是什么？

还有redis也是（docker run方式也没有），是否使用缓存对于最终客户端使用上的区别是什么

---本人技术小白，请赐教！谢谢

### 这个功能可以解决什么问题？

---本人技术小白，请赐教！谢谢

### 这不是重复的功能请求和 RSS 提案

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求及不是 [新的 RSS 提案](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_zh.yml)。

**Comment by TonyRL** [comment]
https://docs.rsshub.app/install

If the route(s) you want to use depends on puppetter then you need to use either `browserless/chrome` or `chromium-bundled` tagged RSSHub image.
